components/clp-package-utils/clp_package_utils/scripts/decompress.py

```diff
@@ -156,6 +161,11 @@ def handle_extract_stream_cmd(
     if clp_config is None:
         return -1
 
+    storage_type = clp_config.archive_output.storage.type
+    if StorageType.FS != storage_type:
+        logger.error(f"Stream extraction is not supported for storage type: {storage_type}.")
+        return -1
```

Comment:
Can we add this to the search script as well? Technically we're going to implement it soon, but better to have a working package for now until that happens.